aws-redshift-clusterparametergroup/src/test/java/software/amazon/redshift/clusterparametergroup/TestUtils.java

```diff
@@ -119,9 +108,25 @@ public class TestUtils {
             software.amazon.awssdk.services.redshift.model.Tag.builder().key("stackKey").value("stackValue").build()
     );
 
-    final static List<String> SDK_TAG_KEYS_TO_DELETE = ImmutableList.of("key3");
-
     final static DescribeTagsResponse DESCRIBE_TAGS_RESPONSE_CREATING = DescribeTagsResponse.builder()
             .taggedResources(TAGGED_RESOURCES_CREATING)
             .build();
+
+    final static boolean parametersEquals(
+            List<software.amazon.awssdk.services.redshift.model.Parameter> params1,
+            List<software.amazon.awssdk.services.redshift.model.Parameter> params2
+    ) {
+        if (params1.size() != params2.size()) {
+            return false;
+        }
+
+        Function<List<software.amazon.awssdk.services.redshift.model.Parameter>, Set<String>> paramsToSet = list -> list.stream()
```

Comment:
Same

Reply:
actually can't in this case, there's another Parameter in this package, with less fields than the awssdk Parameter, which confuses my compiler